test/src/tests-identity.js

```diff
@@ -1030,32 +1009,35 @@ describe('identity', function() {
 
         mParticle.init(apiKey, window.mParticle.config);
 
-        mockServer.requests = [];
-        mParticle.logEvent('Test1');
-        var event1 = getEvent(mockServer.requests, 'Test1');
-        Should(event1).not.be.ok();
+        window.fetchMock._calls = [];
+        mParticle.logEvent('Test Event1');
 
+        var testEvent1 = findEventFromRequest(window.fetchMock._calls, 'Test Event1');
+        
+        Should(testEvent1).not.be.ok();
```

Comment:
I want us to stop using `Should` in favor of `expect`. I think it would make it easier for us to migrate to Jest.

```suggestion
        expect(testEvent1).not.be.ok;
```

Reply:
will consider this out of scope.  sustained engineering ticket - there seem to be issues importing/requireing `expect` into the requirejs tests, so that will require (pun intended) some investigation